importer/EsEs.py

```diff
@@ -0,0 +1,135 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class EsEs(Monument):
+
+    def set_heritage_with_date(self):
+        heritage = self.mapping["heritage"]["item"]
+        if self.has_non_empty_attribute("fecha"):
+            # 20 de febrero de 1985
+            months = ["enero", "febrero", "marzo",
+                      "abril", "mayo", "junio",
+                      "julio", "agosto",
+                      "septiembre", "octubre",
+                      "noviembre", "diciembre", ]
+            date_parts = self.fecha.split(" ")
+            if len(date_parts) != 5:
+                self.add_to_report("fecha", self.fecha)
+                self.add_statement("heritage_status", heritage)
+                return
+            else:
+                day = str(date_parts[0])
+                month = str(months.index(date_parts[2]) + 1)
+                year = str(date_parts[4])
+                date_string = "{}-{}-{}".format(year,
+                                                month.zfill(2), day.zfill(2))
+                try:
+                    date_dict = utils.date_to_dict(date_string, "%Y-%m-%d")
+                except ValueError:
+                    self.add_to_report("fecha", self.fecha)
+                    self.add_statement("heritage_status", heritage)
+                    return
+                qualifier = {"start_time": {"time_value": date_dict}}
+                self.add_statement("heritage_status", heritage, qualifier)
+        else:
+            self.add_statement("heritage_status", heritage)
+
+    def set_adm_location(self):
+        adm_q = None
+        municip_dic = self.data_files["municipalities"]
+        province_dic = self.data_files["provinces"]
+        if utils.count_wikilinks(self.municipio) == 1:
+            adm_q = utils.q_from_first_wikilink("es", self.municipio)
+        else:
+            # sometimes they're in ''
+            municip_raw = utils.remove_markup(self.municipio)
+            municip_match = [x for x in municip_dic if x[
+                "itemLabel"] == municip_raw]
+            if len(municip_match) == 1:
```

Comment:
set muni report as an else here instead. We want it reported even if we fall back on a province

Reply:
Done.